cpp/src/io/csv/csv_reader.cu

```diff
@@ -386,17 +391,17 @@ gdf_error setRecordStarts(const char *h_data, size_t h_size, raw_csv_t *raw_csv)
 }
 
 /**---------------------------------------------------------------------------*
- * @brief Read in a CSV file, extract all fields and return 
- * a GDF (array of gdf_columns)
+ * @brief Reads CSV-structured data and returns an array of gdf_columns.
  *
- * @param[in,out] args Structure containing both the the input arguments 
- * and the returned data
+ * @param[in,out] args Structure containing input and output args
  *
- * @return gdf_error
+ * @return gdf_error GDF_SUCCESS if successful, otherwise an error code.
  *---------------------------------------------------------------------------**/
 gdf_error read_csv(csv_read_arg *args)
 {
-	gdf_error error = gdf_error::GDF_SUCCESS;
+  gdf_error error = gdf_error::GDF_SUCCESS;
+
+  std::vector<gdf_column_wrapper> columns;
```

Comment:
I would prefer this to be declared just before it's used.

Reply:
Done, thanks!